algorithmic_efficiency/spec.py

```diff
@@ -111,7 +111,9 @@ def build_input_queue(self,
                         global_batch_size: int,
                         cache: Optional[bool] = None,
                         repeat_final_dataset: Optional[bool] = None,
-                        num_batches: Optional[int] = None) -> Dict[str, Any]:
+                        num_batches: Optional[int] = None,
+                        use_mixup: bool = False,
+                        mixup_alpha: float = 0.1) -> Dict[str, Any]:
```

Comment:
(I mention it also below, but) I think we should remove the changes to build_input_queue, because we will always run with a fixed mixup_alpha=0.2 only for imagenet_vit, and submitters aren't allowed to tune it. then we can just pass use_mixup=true only for the training split (see below comments)